src/app-gocardless/banks/american-express-aesudef1.js

```diff
@@ -30,7 +32,7 @@ export default {
   },
 
   sortTransactions(transactions = []) {
-    return sortByBookingDateOrValueDate(transactions);
+    return Fallback.sortTransactions(transactions);
   },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏might be easier to remove all these defaults entirely and then just use the object spready syntax. WDYT?

```
export default {
  ...Fallback,
  
   sortTransactions() {
       // custom logic
   },
}
```

Reply:
Could definitely do that.

And then if a new method has to be added down the road, everything just grabs it by default from the Fallback and the banks do not need updated.

Reply:
Yup. That's another benefit (that I definitely thought of before 😄).